lib/matplotlib/tests/test_patheffects.py

```diff
@@ -132,3 +134,61 @@ def test_collection():
                                                        linewidth=3)])
         text.set_bbox({'boxstyle': 'sawtooth', 'facecolor': 'none',
                        'edgecolor': 'blue'})
+
+
+@image_comparison(['tickedstroke'], remove_text=True, extensions=['png'])
+def test_tickedstroke():
+    ax1 = plt.subplot(221)
+    path = Path.unit_circle()
+    patch = patches.PathPatch(path, facecolor='none', lw=2, path_effects=[
+        path_effects.withTickedStroke(angle=-90, spacing=10,
+                                      length=1)])
+
+    ax1.add_patch(patch)
+    ax1.axis('equal')
+    ax1.set_xlim(-2, 2)
+    ax1.set_ylim(-2, 2)
+
+    ax2 = plt.subplot(222)
+    p1, = ax2.plot([0, 1], [0, 1],
```

Comment:
Same as "Lines with a ticked patheffect". Please copy code from there (no need for fancy legend acrobatics). But please use `label=' '` to prevent possible subpixel rendering changes across freetype versions.

Reply:
Should be reconciled in 29fa9c35.  Any remaining differences seem to have a good reason for being.